bindings/nodejs/examples/exchange/1-generate-mnemonic.js

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
Are you trying to trigger me with these 1-indexed examples? Because it's working :trollface:

Reply:
Why shouldn't it? :D

Reply:
I just updated the old guide which also started with 1 :trollface: